### PR TITLE
#335: Fixed DNS resolution in ITDE when running jupyter notebook tests

### LINKED ISSUE #335: DNS resolution in ITDE not working in docker-db
DNS resolution is not working on Github tests, see
https://github.com/exasol/ai-lab/actions/runs/11230524435/job/31613541033

See also https://github.com/exasol/notebook-connector/issues/139

### PR DESCRIPTION
**Pull request by tomuben**
fixes #335

### CODE REVIEW

File: test/notebook_test_runner/test_notebooks_in_dss_docker_image.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def test_notebook(notebook_test_container_with_log, notebook_test_file, notebook
     virtual_env = exec_command(command_echo_virtual_env, container)
     command_run_test = (
         f"{virtual_env}/bin/python"
-        f" -m pytest --setup-show -s --backend={notebook_test_backend} {notebook_test_file}"
+        f" -m pytest --setup-show -s --backend={notebook_test_backend} --itde-nameserver='8.8.8.8' --{notebook_test_file}"
```
Comment (tomuben) [inline test/notebook_test_runner/test_notebooks_in_dss_docker_image.py:84]: Not sure if this is the correct way

Comment (tomuben) [reply test/notebook_test_runner/test_notebooks_in_dss_docker_image.py:84]: Ok, according to the tests this should be ok:
https://github.com/exasol/pytest-plugins/blob/f158045b3045f9065d9d8ce0c16aad3f3806d941/pytest-backend/test/integration/pytest_backend_test.py#L67

File: test/notebook_test_runner/test_notebooks_in_dss_docker_image.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def test_notebook(notebook_test_container_with_log, notebook_test_file, notebook
     virtual_env = exec_command(command_echo_virtual_env, container)
     command_run_test = (
         f"{virtual_env}/bin/python"
-        f" -m pytest --setup-show -s --backend={notebook_test_backend} {notebook_test_file}"
+        f" -m pytest --setup-show -s --backend={notebook_test_backend} --itde-nameserver='8.8.8.8' {notebook_test_file}"
```
Comment (ahsimb) [inline test/notebook_test_runner/test_notebooks_in_dss_docker_image.py:84]: It looks like the right thing to do for me, but how did it work before, without `--itde-namserver` specified explicitly?

**Review by ahsimb — APPROVED** [review]

File: test/notebook_test_runner/test_notebooks_in_dss_docker_image.py (hunk 2 above)
Comment (ckunki) [reply test/notebook_test_runner/test_notebooks_in_dss_docker_image.py:84]: > but how did it work before

For this question, we don't have an answer, yet.

**Comment by ckunki** [comment]
1 job still failed: Running TE notebooks onprem.

I started this job again to check for sporadic failure.

File: test/notebook_test_runner/test_notebooks_in_dss_docker_image.py (hunk 2 above)
Comment (ckunki) [inline test/notebook_test_runner/test_notebooks_in_dss_docker_image.py:84]:
```suggestion
        f"{virtual_env}/bin/python"
        f" -m pytest --setup-show -s"
        f" --backend={notebook_test_backend}"
        f" --itde-nameserver=8.8.8.8"
        f" {notebook_test_file}"
```

**Review by ckunki — DISMISSED** [review]

**Comment by ckunki** [comment]
@tkilias said that the current fail of the Running TE notebooks onprem could be related to too less RAM of the DB (currently 2 GB). 
Proposal add CLI option to increase ITDE RAM size.

File: test/notebook_test_runner/test_notebooks_in_dss_docker_image.py (hunk 2 above)
Comment (tomuben) [reply test/notebook_test_runner/test_notebooks_in_dss_docker_image.py]: Done

**Comment by ckunki** [comment]
Currently, some of the sample data is not available, as well as some of the links in the Jupyter notebooks are broken.

E.g. "https://archive.ics.uci.edu/static/public/1/abalone.zip shows 
> HTTP Error 502: Bad Gateway

**Pull request merged**